scripts/setup/setup.sh

```diff
@@ -25,17 +25,22 @@ if ! [ -x "$(command -v git)" ]; then
 fi
 if ! [ -x "$(command -v node)" ]; then
   echo Error: node is not installed.
-  echo Please Install NodeJS from: https://nodejs.org/en/download/
+  echo Please Install NodeJS from: https://nodejs.org/en/download
   echo And make sure its in the path
   exit 1
 fi
 if ! [ -x "$(command -v npm)" ]; then
   echo 'Error: npm is not installed.' >&2
-  echo Please install npm from: https://nodejs.org/en/download/
+  echo Please install npm from: https://nodejs.org/en/download
   echo And make sure its in the path
   exit 1
 fi
-echo ✓ Dependencies are already Installed
+if ! [ -x "$(command -v pnpm)" ]; then
+  echo PNPM is not installed but he is optionnal
```

Comment:
do not mention that the user can optionally install pnpm, just use pnpm if the user has it installed